.gitignore

```
@@ -0,0 +1 @@
+.env
```

Pedantic missing EOF 
```suggestion
.env

```